server.js

```diff
@@ -0,0 +1,30 @@
+const req = require('request');
+const http = require('http');
+
+ http.createServer(function(request, response) {    
+    if (request.method =='POST'){       
+         var body = '';
+        console.log('Status Code:', response.statusCode);
+        request.on('data', function(data){            
+             body += data;
+        });
+        request.on('end', function(){
+            var d = body;        
+        response.write(d);
+        response.writeHead(200, {"Content-Type": "text/plain"});
+        response.end();                       
+            var options = {
+                method:'post',
+                    url: 'https://hook.integromat.com/15ani2p9j8pub4s7hdwaippiqqrqobvp',
+                    json: true,
+                    headers: {
+                        'Content-Type': 'application/json',
```

Comment:
can you just make sure you're applying a formatting prettifyer when you save this doc?

Reply:
ok